tests/unit/test_install_scripts.py

```
@@ -1076,3 +1076,98 @@ def test_documents_reboot_after_clean(self):
+    The nightly workflow re-runs the install sim against the latest
+    *unpinned* debian:trixie/bookworm/bullseye images so that upstream
+    package churn breaks a nightly job instead of a user's Pi Zero 2 W.
+    These tests exist primarily to prevent the file from being silently
+    deleted — losing this drift detector was the exact class of regression
+    that caused JTN-528 to slip through the whole Trixie release cycle.
+    """
+
+    WORKFLOW_PATH = WORKFLOWS_DIR / "os-drift-nightly.yml"
+
+    @pytest.fixture(autouse=True)
+    def _load(self):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_load`

Add return type annotation: `None`

(ANN202)